frontend/src/components/ambulance/WhyChooseUs.jsx

```
@@ -1,15 +1,74 @@
+
+import React from "react";
+
 import React from 'react';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)